### ISSUE TITLE
example for code-coverage

### ISSUE THREAD

**Issue by karpikpl** [adventworks/nodejs-sample#3]
Hi,
can you add an example for code coverage results displayed in VSTS?

**Comment by lnxlalith** [comment]
Yes! I also want to see the code coverage results in VSTS via sonarqube, I tried it but it shows 0%
